puma/histogram.py

```diff
@@ -435,12 +452,20 @@ def plot(self, **kwargs):
             elem.bin_edges, elem.hist, elem.unc, elem.band = hist_w_unc(
                 elem.values,
                 weights=elem.weights,
+                bin_edges=elem.bin_edges,
+                sum_squared_weights=elem.sum_squared_weights,
                 bins=self.bins,
+                filled=elem.filled,
                 bins_range=self.bins_range,
                 normed=self.norm,
                 underoverflow=self.underoverflow,
             )
 
+            # MAYBE CHECK HERE THAT self.bins and elem.bin_edges are
+            # equivalent for plotting or throw error!
+            # MAYBE also add a else statement that defaults to throwing an error
+            # for unrecognised hist_type!
+
```

Comment:
I'm definitely in favour of adding extra checks if we can :)

I'm not sure what the `hist_type` one refers to though?

Reply:
`hist_type` was something I removed so I just removed the comment as well! (Now we use the attribute filled `instead`) Thanks !